fusillade-api.yml

```diff
@@ -1151,14 +1151,13 @@ paths:
           application/json:
             schema:
               type: object
+              required:
```

Comment:
https://github.com/HumanCellAtlas/fusillade/issues/386 can close this issue with the changes.